.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     'max-lines-per-function': 'off',
     'no-console': ['warn'],
     complexity: ['off'],
+    'object-property-newline': ['error', { allowAllPropertiesOnSameLine: false }],
```

Comment:
Strangely this is not already existing in kent's or prettier's recommended config. We should add it to universe config as well.